CMake/Findsodium.cmake

```diff
@@ -47,7 +47,7 @@ endif()
 
 ########################################################################
 # UNIX
-if (UNIX OR AMIGA)
+if (UNIX)
```

Comment:
Since Amiga has NONET defined I don't feel there is any reason to try and modify this file

Reply:
The point was to try to get networking running on Amiga in the future as well. But the current network code might be too complex for Amiga anyway. Too much extra dependencies and missing networking code in the OS SDK.